modules/videopress/shortcode.php

```diff
@@ -150,7 +150,7 @@ function videopress_handle_editor_view_js() {
 
 	add_action( 'admin_print_footer_scripts', 'videopress_editor_view_js_templates' );
 
-	wp_enqueue_script( 'videopress-editor-view', plugins_url( 'js/editor-view.js', __FILE__ ), array( 'wp-util', 'jquery' ), false, true );
+	wp_enqueue_script( 'videopress-editor-view', plugins_url( 'js/editor-view.js', __FILE__ ), array( 'wp-util', 'media-editor', 'jquery' ), false, true );
```

Comment:
@Lewiscowles1986 I believe we enqueue the scripts as long as either VideoPress or the Shortcodes module are enabled. Checking for just VideoPress isn't enough, as some folks might want to embed videos on their site, even if they didn't upload the videos.